synapse/http/site.py

```diff
@@ -269,6 +270,11 @@ def _finished_processing(self):
         if authenticated_entity is not None and isinstance(authenticated_entity, bytes):
             authenticated_entity = authenticated_entity.decode("utf-8", "replace")
 
+        if self.target_user:
+            authenticated_entity = "{} as {}".format(
+                authenticated_entity, self.target_user,
+            )
```

Comment:
we're using the `as` format even if the two user ids are the same?

Reply:
Woops, that is a hold over from when `target_user` was None if they were the same. The intention is to only use the format if the two users are different.

Also, I might change the format to be `{<auth_entity>,<target_user>}` to make parsing the logs easier (as we use spaces as separators up to the message)